roles/spark-standalone-install/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+
+  - name: spark | deps | install
+    apt: name={{ item }} state=present update_cache=yes
+    with_items:
+      - python-setuptools
+      - python-dev
+      - python-boto
+      - openjdk-7-jre-headless
+      - openjdk-7-jdk
+      - openssh-client
+      - openssh-server
+      - rsync
+      - sudo
+      - tar
+      - gzip
+      - wget
+      - curl
+      - unzip
+    when: do_install|bool
+
+  - name: spark | service | stop
+    service:
+        name: spark
```

Comment:
This needs to be `spark-dispatcher` as you currently have the role written; `spark-master` if you decide to rename the service.